### DISCUSSION TITLE
error on xubuntu

### DISCUSSION THREAD

**Discussion by Bibo89** [Megghy/MultiSEngine#18] [Q&A]
i got this error any solution?

Error] Failed to send data: LoadNetModule
System.IO.IOException: Unable to write data to the transport connection: Broken pipe.
 ---> System.Net.Sockets.SocketException (32): Broken pipe
   at System.Net.Sockets.NetworkStream.Write(ReadOnlySpan`1 buffer)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Write(ReadOnlySpan`1 buffer)
   at System.IO.BinaryWriter.Write(ReadOnlySpan`1 buffer)
   at MultiSEngine.DataStruct.TcpContainer.Send(Span`1& buf) in /home/server/MultiSEngine/MultiSEngine/DataStruct/SocketContainer.cs:line 67

**Comment by Bibo89** [comment]
[Error] Failed to send data: Disconnect
System.IO.IOException: Unable to write data to the transport connection: Connessione interrotta dal software del computer host..
 ---> System.Net.Sockets.SocketException (10053): Connessione interrotta dal software del computer host.
   at System.Net.Sockets.NetworkStream.Write(ReadOnlySpan`1 buffer)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Write(ReadOnlySpan`1 buffer)
   at System.IO.BinaryWriter.Write(ReadOnlySpan`1 buffer)
   at MultiSEngine.DataStruct.TcpContainer.Send(Span`1& buf) in D:\Code\MultiSEngine\MultiSEngine\DataStruct\SocketContainer.cs:line 67


this on windows,
work but item cloned throug servers

**Comment by Megghy** [comment]
Can you show me your configuration file? This program works fine in my own linux environment

**Reply by Bibo89** [reply]
hi, I'll explain the problem, the connection to multiSEngine practically works but when I change the server it gives me the error. then if I do several times I change the server characters objects are mixed. and if I keep it active for a long time then it doesn't connect to multisengine anymore.



{
  "ListenIP": "0.0.0.0",
  "ListenPort": 7778,
  "ServerName": "LOBBY",
  "ServerVersion": 279,
  "Token": "114514",
  "SwitchTimeOut": 10000,
  "EnableCrossplayFeature": true,
  "EnableChatForward": true,
  "ChatFormat": "[{servername}] {username}: {message}",
  "SwitchToDefaultServerOnJoin": true,
  "RestoreDataWhenJoinNonSSC": true,
  "DefaultServer": "SURVIVAL",
  "Servers": [
    {
      "Visible": true,
      "Name": "SURVIVAL",
      "ShortName": "sur",
      "IP": "54.36.234.246",
      "Port": 7777,
      "SpawnX": 3800,
      "SpawnY": 488,
      "VersionNum": -1
    },
    {
      "Visible": true,
      "Name": "CREATIVE - New Open!!",
      "ShortName": "cre",
      "IP": "127.0.0.1",
      "Port": 7777,
      "SpawnX": 3203,
      "SpawnY": 439,
      "VersionNum": -1
    }
  ]
}
